Flaky test - fluentforwardreceiver TestBodyConversion

Seen here: https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/4044206024?check_suite_focus=true

```
make -C ./receiver/fluentforwardreceiver test
make[2]: Entering directory '/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/fluentforwardreceiver'
go test -race -timeout 300s --tags="" ./...
gzip: invalid header--- FAIL: TestBodyConversion (0.00s)
    conversion_test.go:269: 
        	Error Trace:	conversion_test.go:269
        	Error:      	Not equal: 
        	            	expected: pdata.LogRecord{orig:(*v1.LogRecord)(0xc000178180)}
        	            	actual  : pdata.LogRecord{orig:(*v1.LogRecord)(0xc000178080)}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -10,10 +10,2 @@
        	            	      Values: ([]v1.KeyValue) (len=3) {
        	            	-      (v1.KeyValue) {
        	            	-       Key: (string) (len=1) "a",
        	            	-       Value: (v1.AnyValue) {
        	            	-        Value: (*v1.AnyValue_StringValue)({
        	            	-         StringValue: (string) (len=5) "value"
        	            	-        })
        	            	-       }
        	            	-      },
        	            	       (v1.KeyValue) {
        	            	@@ -39,2 +31,10 @@
        	            	          })
        	            	+        })
        	            	+       }
        	            	+      },
        	            	+      (v1.KeyValue) {
        	            	+       Key: (string) (len=1) "a",
        	            	+       Value: (v1.AnyValue) {
        	            	+        Value: (*v1.AnyValue_StringValue)({
        	            	+         StringValue: (string) (len=5) "value"
        	            	         })
        	Test:       	TestBodyConversion
[{{debug 2021-10-29 08:20:51.772283621 +0000 UTC m=+0.222846789  Got connection undefined } [{remoteAddr 15 0 127.0.0.1:51022 <nil>}]}]FAIL
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver	0.539s
```

--- a/receiver/fluentforwardreceiver/conversion_test.go
+++ b/receiver/fluentforwardreceiver/conversion_test.go
@@ -19,6 +19,7 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/tinylib/msgp/msgp"
 	"go.opentelemetry.io/collector/model/pdata"
@@ -266,7 +267,10 @@ func TestBodyConversion(t *testing.T) {
 	cv.MapVal().InsertInt("d", 24)
 	body.MapVal().Insert("c", cv)
 
-	require.EqualValues(t, Logs(
+	// Sort the map, sometimes may get in a different order.
+	require.Equal(t, pdata.AttributeValueTypeMap, le.Body().Type())
+	le.Body().MapVal().Sort()
+	assert.EqualValues(t, Logs(
 		Log{
 			Timestamp: 5000000000000,
 			Body:      body,

